code/drasil-code/lib/Language/Drasil/Code/Imperative/GOOL/LanguageRenderer.hs

```diff
@@ -1,20 +1,22 @@
 module Language.Drasil.Code.Imperative.GOOL.LanguageRenderer (
   -- * Common Syntax
-  doxConfigName, makefileName, sampleInputName, readMeName
+  doxConfigName, makefileName, sampleInputName, sampleOutputName, readMeName
 ) where
 
 ----------------------------------------
 -- Syntax common to several renderers --
 ----------------------------------------
 
 -- | Common syntax for several renderers.
-doxConfigName, makefileName, sampleInputName, readMeName :: String
+doxConfigName, makefileName, sampleInputName, sampleOutputName, readMeName :: String
```

Comment:
see issue #3490. Note that this shouldn't be fixed in this PR, but a subsequent one.